docs/type-definitions/object-types.md

```diff
@@ -58,14 +58,15 @@ This example uses **inline** style for Object Type definitions, but you can also
+| name         | `string`               | **Required.** Unique name of this object type within Schema                                                                                                                                                                                                                                                                                                                                |
+| fields       | `array` or `callable`  | **Required**. An array describing object fields or callable returning such an array. See [field configuration options](#field-configuration-options) section below for expected structure of each array entry. See also the section on [Circular types](#recurring-and-circular-types) for an explanation of when to use callable for this option.                                         |
+| description  | `string`               | Plain-text description of this type for clients (e.g. used by [GraphiQL](https://github.com/graphql/graphiql) for auto-generated documentation)                                                                                                                                                                                                                                            |
+| interfaces   | `array` or `callable`  | List of interfaces implemented by this type or callable returning such a list. See [Interface Types](interfaces.md) for details. See also the section on [Circular types](#recurring-and-circular-types) for an explanation of when to use callable for this option.                                                                                                                       |
+| isTypeOf     | `callable`             | **function ($value, $context, [ResolveInfo](../class-reference.md#graphqltypedefinitionresolveinfo) $info): bool**<br> Expected to return **true** if **$value** qualifies for this type (see section about [Abstract Type Resolution](interfaces.md#interface-role-in-data-fetching) for explanation).                                                                                    |
+| resolveField | `callable`             | **function ($value, array $args, $context, [ResolveInfo](../class-reference.md#graphqltypedefinitionresolveinfo) $info): mixed**<br> Given the **$value** of this type, it is expected to return value for a field defined in **$info->fieldName**. A good place to define a type-specific strategy for field resolution. See section on [Data Fetching](../data-fetching.md) for details. |
+| visible      | `bool` or `callable`   | **function ($context): mixed**<br> You can customize the visibility of fields by defining them as visible or not visible using a `bool` or your own custom logic by sending a `callable` that receives the `$context`. In introspection, the field will not be included in the result and if a query references that field, it will return a validation error.                             |
```

Comment:
> that receives the `$context`

As mentioned in https://github.com/webonyx/graphql-php/pull/1434#issuecomment-1735062027, that is no longer the case.

```suggestion
| visible      | `bool` or `callable`   | Defaults to `true`. The given callable receives no arguments and is expected to return a `bool`, it is called once  when the field may be accessed. The field is treated as if it were not defined at all when this is `false`. |
```